src/viewBase.js

```diff
@@ -84,6 +111,22 @@ export default class ViewBase extends HasModels {
    * The prefix to use for the templates. Defaults to `view-<model>-`
    * @return {string}
    */
+  listTemplate() {
+    return this.opts.listTemplate || "page"
+  }
+
+  /**
+   * The prefix to use for the templates. Defaults to `view-<model>-`
```

Comment:
These doc strings seem incorrect, but also these methods aren't really for the "template" but the ... parent template / layout, right?